src/util.ts

```diff
@@ -31,3 +33,22 @@ export function sortBy<T>(arr: T[], fn: (i: T) => sort.Types | sort.Types[]): T[
 
   return arr.sort((a, b) => compare(fn(a), fn(b)))
 }
+
+export function getHelpPluginPackage(pjson: Config['pjson'], defaultPlugin = '@oclif/plugin-help'): string {
+  const configuredPlugin = pjson.oclif.helpPlugin
+
+  if (configuredPlugin) {
+    try {
+      require(configuredPlugin)
+      return configuredPlugin
+    } catch {}
+  }
+
+  try {
+    require(defaultPlugin)
+    return defaultPlugin
+  } catch {}
+
+  if (configuredPlugin) throw new Error(`Unable to load configured help plugin "${configuredPlugin}" from package.json`)
```

Comment:
Can we put this in the catch and also append the actual error message. I would actually fine this very frustrating to set my plugin-help during development but getting default behaviour and not knowing why because it swallows my error.